docs/deployment/containers/kubernetes.rst

```diff
@@ -291,6 +291,10 @@ CrateDB 3.0.5 cluster:
               value: "3"
             - name: CLUSTER_NAME
               value: "my-crate"
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
```

Comment:
Yep, that should work. On the cloud-stack we use .... 

```
        - name: POD_NAME
          valueFrom:
            fieldRef:
              apiVersion: v1
              fieldPath: metadata.name
```

Reply:
@WalBeh is it desirable to include `apiVersion` in the docs example?